src/helpers/tokenAuthentication.js

```diff
@@ -0,0 +1,37 @@
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+
+class TokenAuthenticate {
+    static generateToken = (userDetails, expires) => {
+        const token = jwt.sign(userDetails,
+            process.env.SECRET, { expiresIn: expires }
+        );
+    return token;
+    }
+
+    static async tokenVerify (req, res, next) {
+        const token = req.headers['Authorization'] || req.headers['x-access-token'] || req.query.token || req.body.token || req.params.token;
```

Comment:
I'm not sure you need this `req.params.token`. I've not seen a case where you'll pass a token as a param. If you think you'll need it, fine. If not, you should remove it.

Reply:
It was just a thought. If you suggest so I'd remove it.